src/components/DateField.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import DateFnsUtils from '@date-io/date-fns';
+import 'date-fns';
+import {
+  MuiPickersUtilsProvider,
+  KeyboardDatePicker,
+} from '@material-ui/pickers';
+import PropTypes from 'prop-types';
+
+function DateField(props) {
+  const { id, label, value, handler } = props;
```

Comment:
I suggest taking in the onChange function directly. The caller already knows what the id is because they passed that in